### DISCUSSION TITLE
Anyone can help me ? i get the Ticking world crash on my modpack.

### DISCUSSION THREAD

**Discussion by Kirinat0** [FabricMC/fabric-api#1621] [FAQ] [answered]
here the crash report

---- Minecraft Crash Report ----
// Don't do that.

Time: 8/9/21 9:41 AM
Description: Exception ticking world

java.lang.NullPointerException: Exception ticking world
	at com.biom4st3r.moenchantments.EnchantmentRegistry.lambda$static$22(EnchantmentRegistry.java:106)
	at com.biom4st3r.moenchantments.EnchantmentRegistry$$Lambda$4945/1041729529.apply(Unknown Source)
	at biom4st3r.libs.moenchant_lib.EnchantmentSkeleton.applicationLogic(EnchantmentSkeleton.java:255)
	at net.minecraft.class_1890.handler$zim000$re_add_moenchntments(class_1890.java:1531)
	at net.minecraft.class_1890.method_8229(class_1890.java:398)
	at net.minecraft.class_1890.method_8230(class_1890.java:337)
	at net.minecraft.class_1890.method_8233(class_1890.java:295)
	at net.minecraft.class_1308.method_30759(class_1308.java:1059)
	at net.minecraft.class_1308.method_5984(class_1308.java:1047)
	at net.minecraft.class_1547.method_5943(class_1547.java:153)
	at net.minecraft.class_1948.method_24930(class_1948.java:279)
	at net.minecraft.class_1948.method_8663(class_1948.java:197)
	at net.minecraft.class_1948.method_27821(class_1948.java:185)
	at net.minecraft.class_3215.method_20801(class_3215.java:388)
	at net.minecraft.class_3215$$Lambda$9220/1812500663.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at net.minecraft.class_3215.method_14161(class_3215.java:368)
	at net.minecraft.class_3215.method_12127(class_3215.java:337)
	at net.minecraft.class_3218.method_18765(class_3218.java:353)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:871)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:811)
	at net.minecraft.class_1132.method_3748(class_1132.java:91)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at net.minecraft.server.MinecraftServer$$Lambda$8617/1498047314.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at com.biom4st3r.moenchantments.EnchantmentRegistry.lambda$static$22(EnchantmentRegistry.java:106)
	at com.biom4st3r.moenchantments.EnchantmentRegistry$$Lambda$4945/1041729529.apply(Unknown Source)
	at biom4st3r.libs.moenchant_lib.EnchantmentSkeleton.applicationLogic(EnchantmentSkeleton.java:255)
	at net.minecraft.class_1890.handler$zim000$re_add_moenchntments(class_1890.java:1531)
	at net.minecraft.class_1890.method_8229(class_1890.java:398)
	at net.minecraft.class_1890.method_8230(class_1890.java:337)
	at net.minecraft.class_1890.method_8233(class_1890.java:295)
	at net.minecraft.class_1308.method_30759(class_1308.java:1059)
	at net.minecraft.class_1308.method_5984(class_1308.java:1047)
	at net.minecraft.class_1547.method_5943(class_1547.java:153)
	at net.minecraft.class_1948.method_24930(class_1948.java:279)
	at net.minecraft.class_1948.method_8663(class_1948.java:197)
	at net.minecraft.class_1948.method_27821(class_1948.java:185)
	at net.minecraft.class_3215.method_20801(class_3215.java:388)
	at net.minecraft.class_3215$$Lambda$9220/1812500663.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at net.minecraft.class_3215.method_14161(class_3215.java:368)
	at net.minecraft.class_3215.method_12127(class_3215.java:337)
	at net.minecraft.class_3218.method_18765(class_3218.java:353)

-- Affected level --
Details:
	All players: 1 total; [class_3222['LuBlade'/168, l='ServerLevel[Blade]', x=-2921.10, y=72.00, z=-84.47]]
	Chunk stats: ServerChunkCache: 3250
	Level dimension: minecraft:overworld
	Level spawn location: World: (16,69,-160), Chunk: (at 0,4,0 in 1,-10; contains blocks 16,0,-160 to 31,255,-145), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 563227 game time, 798910 day time
	Level name: Blade
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Level weather: Rain time: 78611 (now: false), thunder time: 33994 (now: false)
	Known server brands: fabric
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:871)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:811)
	at net.minecraft.class_1132.method_3748(class_1132.java:91)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at net.minecraft.server.MinecraftServer$$Lambda$8617/1498047314.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 853409760 bytes (813 MB) / 2423259136 bytes (2311 MB) up to 4235722752 bytes (4039 MB)
	CPUs: 4
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4544m -Xms256m -XX:PermSize=256m
	Fabric Mods: 
		adorn: Adorn 1.14.1+1.16.5
		advanced_runtime_resource_pack: Runtime Resource Pack 0.2.9
		angerable-patch: Angerable Patch 1.0.0-1.16.1
		antighost: AntiGhost 1.16.5-fabric0.30.0-1.1.3
		appleskin: AppleSkin mc1.16.4-2.1.0
		appliedenergistics2: Applied Energistics 2 8.4.0-alpha.2
		architects_palette: Architect's Palette Fabric 1.0
		architectury: Architectury 1.20.29
		artofalchemy: Art of Alchemy: Memoriam 1.1.0+1.16.5
		attributefix: Attribute Fix {FABRIC} 1.0.2
		autoconfig1u: Auto Config v1 Updated 3.3.1
		autoconfigtoml: AutoConfig-TOML 1.0.0-fabric
		bambootweaks: Bamboo Tweaks 2.0.1+1.16.5
		battletowers: Battle Towers 3.3.4-1.16.4
		bclib: BCLib 0.1.45
		better_withered_mobs: Better Withered Mobs 1.5.2
		betterbeds: Better Beds 1.1.0
		bettercaves: YUNG's Better Caves 1.16.5-1.2
		betterend: Better End 0.9.8.5-pre
		bettermineshafts: YUNG's Better Mineshafts 1.16.4-1.0.2
		betternether: Better Nether 5.0.7
		betterstrongholds: YUNG's Better Strongholds 1.16.5-1.1.1
		biom4st3r_repackaged_expressions: expj4_repackaged 0.1.0
		blockus: Blockus 2.1.10+1.16.5
		blue_endless_jankson: jankson 1.2.0
		bno: Basic Nether Ores 1.16.5-5.5.0-Fabric
		buildguide: Build Guide 0.2.3
		byg: BYG 1.1.8
		can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.0.2
		capes: Capes 1.1.2
		cardinal-components-base: Cardinal Components API (base) 2.8.2
		cardinal-components-block: Cardinal Components API (blocks) 2.7.9
		cardinal-components-chunk: Cardinal Components API (chunks) 2.8.2
		cardinal-components-entity: Cardinal Components API (entities) 2.8.2
		cardinal-components-item: Cardinal Components API (items) 2.7.2
		cardinal-components-world: Cardinal Components API (worlds) 2.8.2
		carrier: Carrier 1.7.6
		cavebiomes: Cave Biomes 0.6.2
		charm: Charm 2.3.2
		cleancut: CleanCut mc1.16-2.1-fabric
		cloth-api: Cloth API 1.4.9
		cloth-armor-api-v1: Cloth Armor API v1 1.4.9
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.5.47
		cloth-common-events-v1: Cloth Common Events v1 1.4.9
		cloth-config2: Cloth Config v4 4.11.26
		cloth-datagen-api-v1: Cloth Datagen v1 1.4.9
		cloth-durability-bar-api-v1: Cloth Durability Bar API v1 1.4.9
		cloth-dynamic-registry-api-v1: Cloth Dynamic Registry API v1 1.4.9
		cloth-scissors-api-v1: Cloth Scissors API v1 1.4.9
		cloth-utils-v1: Cloth Utils v1 1.4.9
		com_electronwill_night-config_core: core 3.6.3
		com_electronwill_night-config_toml: toml 3.6.3
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		conjuring: Conjuring 0.2.5
		consistency_plus: Consistency Plus 0.3.0
		cosmetic-armor: Cosmetic Armor 1.0.0
		cotton-client-commands: Cotton Client Commands 1.0.1+1.16-rc1
		craftable_enchanted_golden_apple: Craftable Enchanted Golden Apple 1.0.0
		craftnstar: Craftable nether star 1.1.0
		croptopia: Croptopia 1.4.0
		croptosis: Croptosis 1.1.0
		crowdin-translate: CrowdinTranslate 1.2
		curios: Curios API 0.0.13-1.16.5
		customportalapi: Custom Portals 0.0.1-beta32-1.16
		dark-enchanting: Dark Enchanting 0.2.1+mc.1.16.5
		diggusmaximus: Diggus Maximus 1.4.2-1.16.3
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.3
		dynamicfps: Dynamic FPS 2.0.2
		easierchests: EasierChests 1.16.5-fabric0.30.0-1.6.3
		easiervillagertrading: EasierVillagerTrading 1.16.5-fabric0.30.0-1.5.4
		emerald_tools: Emerald Tools 1.1.3
		enchant_giver: Enchant Giver 1.0.3
		enchdesc: EnchantmentDescriptions 2.0.8
		entityculling: EntityCulling-Fabric 1.3.0
		expandedstorage: Expanded Storage 7.0.0
		ezpas: Ezpas 1.2.3-1.16.2
		fabric: Fabric API 0.37.1+1.16
		fabric-api-base: Fabric API Base 0.3.0+c88702897d
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.2.0+6aebe11a7d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.10+3661eaa17d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.2+95b14da07d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.6+61336ba97d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.1.0+c88702897d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.3+f941baf07d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.7+2868a2287d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-language-kotlin: Fabric Language Kotlin 1.6.1+kotlin.1.5.10
		fabric-language-scala: Fabric Language Scala 1.1.0+scala.2.13.6
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.3.0+bc7a746f7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.3+e3c9d0627d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.5+c88702897d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.4+f3e8589e7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.3.0+2868a2287d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.6.0+2868a2287d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.7+6afaa13c7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.11+ca0ef93d7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.7+8183d3957d
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.0.0+5f6b94517d
		fabricloader: Fabric Loader 0.11.6
		fallingtree: FallingTree 2.11.0
		fasttransferlib: Fast Transfer Lib 0.4.2
		ferritecore: FerriteCore 2.0.3
		fiber: fiber 0.23.0-2
		fluidtank: Fluid Tank 16.1.1-fabric
		fpsreducer: FPS Reducer 1.21-mc1.16.5
		gbfabrictools: GBfabrictools 1.3.2+1.16.4
		go-fish: Go Fish 1.2.3-beta-1.16.5
		gobber2: Gobber2 2.3.74
		golemsgalore: Golems Galore 0.2.1+mc.1.16.5
		gravestones: Gravestones v1.9
		inventoryhud: Inventory HUD + 3.3.0
		iris: Iris 1.1.1
		ironfurnaces: Iron Furnaces 2.4.1
		jankson: Jankson 3.0.1+j1.2.0
		java: Java HotSpot(TM) 64-Bit Server VM 8
		kibe: Kibe 1.7-BETA
		kyrptconfig: Kytpt Config 1.1.4-1.16
		lambdynlights: LambDynamicLights 1.3.4+1.16
		lazydfu: LazyDFU 0.1.2
		lchunkloader: Literally Chunk Loader 2.0.3
		libblockattributes: LibBlockAttributes 0.8.8
		libblockattributes_core: LibBlockAttributes (Core) 0.8.8
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.8.8
		libblockattributes_items: LibBlockAttributes (Items) 0.8.8
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 3.4.0+1.16.5
		libstructure: LibStructure 1.5
		lithium: Lithium 0.6.6
		magna: Magna 1.5.3-1.16.5
		mannequins: Mannequins 1.0.1
		maybe-data: Maybe data 1.0.0-1.16.2
		mcda: MC Dungeons Armors 1.5.0
		mcdw: MC Dungeons Weapons 3.2.0
		mcsa: MC Story Mode Armor 2.3.2
		minecraft: Minecraft 1.16.5
		mobfarmutilities: Mob Farm Helpers 1.4.3
		modmenu: Mod Menu 1.16.11
		moenchant_lib: Mo'Enchantment Library 0.1.0
		moenchantments: Mo'Enchantments 1.15.1
		morevillagers-fabric: MoreVillagersFabric 1.4.1-SNAPSHOT
		mostructures: Mo' Structures 1.2.0-1.16.5
		multi_item_lib: Multi Item Lib 1.3.0
		mysticalagriculture: Mystical Agriculture: Refabricated 1.1.7
		naturescompass: Nature's Compass 1.16.5-2.0.1-fabric
		netherite_plus: Netherite Plus Mod 1.4.5-1.16.5
		noenchantcap: NoEnchantCap 4.1.0
		omega-config: OmegaConfig 1.0.4
		org_aeonbits_owner_owner: owner 1.0.12
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.5.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.0
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.0
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.2.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.2.1
		org_objenesis_objenesis: objenesis 3.2
		oro_config: OroArmor Config 3.0.0
		owo: OwO 0.1
		particle_emitter_biom4st3r: Particle Emitters LIB 0.1.0
		patchouli: Patchouli 1.16.4-50-FABRIC
		phosphor: Phosphor 0.7.2+build.12
		plantinajar: Plant In A Jar 2.1.2
		playerabilitylib: Pal 1.2.2
		polymorph: Polymorph 0.0.10-1.16.5
		pugh_lib: Pugh Lib 1.1.0
		randompatches: RandomPatches 2.4.5-fabric
		rawore: raworefabric 1.0.0
		reach-entity-attributes: Reach Entity Attribute 1.0.1
		reapingmod: Reaping Mod 1.2.2
		reborncore: Reborn Core 4.7.3+build.136
		resourceful_tools: Resourceful Tools 1.0.56
		rightclickharvest: RightClickHarvest 1.0.0
		ring_of_attraction: Ring Of Attraction 1.0.29
		ring_of_growth: Ring Of Growth 1.0.26
		ringsofascension: Rings of Ascension 1.1.1
		roughlyenoughitems: Roughly Enough Items 5.12.248
		roughlyenoughitems-api: REI (API) 5.12.248
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.12.248
		roughlyenoughitems-runtime: REI (Runtime) 5.12.248
		roughlyenoughresources: Roughly Enough Resources 2.1.5
		simple_backpack: SimpleBackpack 1.0.40
		skinnedlanterns: Skinned Lanterns 1.16.5-1.1.3
		smoothboot: Smooth Boot 1.16.5-1.6.0
		smootherbedrock: Smoother Bedrock 1.0.1
		sodium: Sodium 0.2.0+IRIS3-build.49
		spawnermod: Enhanced Mob Spawners 1.1.1
		spruceui: SpruceUI 2.0.4+1.16
		staffofbuilding: Staff of Building 1.2.6-1.16.5
		static-content: Static Content 1.0.2-1.16.2
		staticdata: Static Data 1.1.2
		structure-helpers: Structure Helpers 3.0.0
		structurized: Structurized 1.2.0-1.16.5
		team_reborn_energy: Energy 0.1.1
		techreborn: Tech Reborn 3.8.4+build.236
		terrarianslimes: Terrarian Slimes 1.1-BETA+1.16
		tk_valoeghese_zoesteriaconfig: ZoesteriaConfig 1.3.6
		trinkets: Trinkets 2.6.7
		truerooms: True Secret Rooms 1.1.3
		vanilla-hammers: Vanilla Hammers 3.0.3-1.16.5
		vexc: Vein Excavator 1.0.0
		voidtotem: Void Totem (Fabric) 1.16.5-1.2.0
		walljump: Wall-Jump! 1.3.3+1.16.1
		waystones: Waystones 2.0.1
		wthit: wthit 2.10.4
		xaerominimap: Xaero's Minimap 21.14.1
		xaeroworldmap: Xaero's World Map 1.16.0
		yungsapi: YUNG's API 1.16.5-Fabric-7
	Patchouli open book context: n/a
	Player Count: 1 / 8; [class_3222['LuBlade'/168, l='ServerLevel[Blade]', x=-2921.10, y=72.00, z=-84.47]]
	Data Packs: vanilla, Fabric Mods
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fabric'

**Comment by warjort** [comment, marked as answer]
Report it to the mod developer:

````
java.lang.NullPointerException: Exception ticking world
at com.biom4st3r.moenchantments.EnchantmentRegistry.lambda$static$22(EnchantmentRegistry.java:106)
````

````
moenchantments: Mo'Enchantments 1.15.1

**Reply by Kirinat0** [reply]
OK Bro thankyou very much
